### ISSUE TITLE
undefined method 'context' when running a spec file

### ISSUE THREAD

**Issue by harshwardhanedu** [ywen/minitest-spec-context#3]
**I am getting this below error when I try to run a spec file**

```
spec/examples/file_name_spec.rb:33:in `block (2 levels) in <main>': undefined method `context' for ClassNamespace:Class (NoMethodError)

 

    context 'context_title' do
    ^^^^^^^
Did you mean?  const_get
        from <ruby_path>/lib/ruby/gems/3.2.0/gems/minitest-5.19.0/lib/minitest/spec.rb:94:in `class_eval'
        from <ruby_path>/lib/ruby/gems/3.2.0/gems/minitest-5.19.0/lib/minitest/spec.rb:94:in `describe'
        from spec/examples/file_name_spec.rb:14:in `block in <main>'
        from <ruby_path>/lib/ruby/gems/3.2.0/gems/minitest-5.19.0/lib/minitest/spec.rb:94:in `class_eval'
        from <ruby_path>/lib/ruby/gems/3.2.0/gems/minitest-5.19.0/lib/minitest/spec.rb:94:in `describe'
        from spec/examples/file_name_spec.rb:5:in `<main>'
        
```
     
**Mentioning the versions in use while getting this issue**

- ruby 3.2.1
- minitest v2.19.0
- minitest-hooks v1.5.1
- minitest-spec-context v0.0.4

**Comment by harshwardhanedu** [comment]
@ywen Can you please have a look at this issue please?

**Comment by ywen** [comment]
Hey @harshwardhanedu 

Please provide a minimum working example. Thanks

**Comment by harshwardhanedu** [comment]
@ywen 

`# spec/my_app_spec.rb

describe "MyApp" do
  context "when performing a calculation" do
    it "returns the result" do
      result = MyApp.perform_calculation(5, 3)
      expect(result).to eq(8)
    end

    it "raises an exception for division by zero" do
      expect { MyApp.perform_calculation(10, 0) }.to raise_error(ZeroDivisionError)
    end
  end
end`


To run this 

rspec spec/my_app_spec.rb

**Comment by harshwardhanedu** [comment]
@ywen Hey! Did you got chance to check on this???

**Comment by skaes** [comment]
The problem is that minitest 5.19 removes the MiniTest constant. You have to use Minitest now.

**Comment by ywen** [comment]
All right, thanks. I will fix it this weekend, cheers

**Comment by ywen** [comment]
Fixed in the new gem version 0.0.5. Thanks fir reporting and debugging

**Comment by harshwardhanedu** [comment]
> Fixed in the new gem version 0.0.5. Thanks for reporting and debugging

@ywen 
Thank you for the fix 😊✔
